content/ActiveAdmin 2FA with OneLogin.md

```diff
@@ -0,0 +1,191 @@
+# **ActiveAdmin 2FA with OneLogin**
+
+![main image](images/2fa.jpg)
+
+Here at Rootstrap, security is one of the most important topics we discuss with our clients. We ensure that hackers and cyber-thieves can’t access any sensitive information from our client’s web apps.
+
+In this blog post, you’ll learn how to configure two factor authentication with OneLogin for your admin panel
+
+
+##
+**1) Some key concepts**
+
+ - SLO (Single log out): SLO is a process that allow users to be logged out in one place and spread it over multiple applications
+
+ - SSO (Single sign in) : SSO is a process that allows users to authenticate into multiple services after logging into a primary service
+
+ - Callback URL : Is a local URL in your app where third party auth provider send you auth data like confirmations or logout requests
+
+
+##
+**2) How to start with all this work?**
+
+
+First take a look at this lovely [gem](https://github.com/apokalipto/devise_saml_authenticatable), you will notice that all the examples are for the `User` class, but don't worry we will show some examples for admin specifically
+
+The specific lines you need to add are:
```

Comment:
Maybe explain a bit more about what each line does. I'd be a bit unsure about adding stuff I don't understand